specification/batch/resource-manager/Microsoft.Batch/stable/2023-05-01/BatchManagement.json

```diff
@@ -6183,8 +6207,8 @@
       "required": [
         "accountName",
         "azureFileUrl",
-        "relativeMountPath",
-        "accountKey"
+        "accountKey",
+        "relativeMountPath"
```

Comment:
Can we preserve this order? I can't remember for certain, but I believe this may affect function signatures in the generated SDKs.

Reply:
Thanks Dave, I changed the order as suggestion.